wic/grub-full.cfg

```diff
@@ -1,6 +1,6 @@
 serial --unit=0 --speed=115200 --word=8 --parity=no --stop=1
 set debug=linux,relocator,multiboot,multiboot_loader,slaunch
-terminal_output console serial
```

Comment:
Removed that commit:
https://github.com/3mdeb/meta-trenchboot/compare/422c3a0387cb197b9df52cf7f5c07674b672415f..beeccbbee447c39a4a9c7da4d6b6b5828da0bd53

It fixes doubled output on APUs because `console` is also serial output there.  I suspected it might break VGA.  Not sure if there is a way to check for `console` being serial at run-time.